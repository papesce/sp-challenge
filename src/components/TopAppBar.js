import React from 'react';
import AppBar from 'material-ui/AppBar';
import Toolbar from 'material-ui/Toolbar';
import Typography from 'material-ui/Typography';


const TopAppBar = () => {
  return (
    <div>
      <AppBar position="static">
        <Toolbar>
          <Typography type="title" color="inherit">
            Spotify Challenge
          </Typography>
         
        </Toolbar>
      </AppBar>
    </div>
  );
}



export default TopAppBar;