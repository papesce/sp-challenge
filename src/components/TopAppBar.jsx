import React from "react";
import AppBar from "material-ui/AppBar";
import Toolbar from "material-ui/Toolbar";
import Typography from "material-ui/Typography";

const TopAppBar = ({ children }) => {
  return (
    <AppBar position="static">
      <Toolbar>
        <Typography type="title" color="inherit">
          Spotify Challenge
        </Typography>
      </Toolbar>
    </AppBar>
  );
};

export default TopAppBar;
