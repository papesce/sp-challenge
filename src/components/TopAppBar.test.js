import React from 'react'
import TopAppBar from './TopAppBar'
import { shallow } from "enzyme";
import renderer from 'react-test-renderer';

describe('SearchBar tests', () => {
    it('should render', () => {  
        shallow(<TopAppBar/>);
    })
    it('renders correctly', () => {
        const tree = renderer.create(<TopAppBar/>)
            .toJSON();
         expect(tree).toMatchSnapshot();
    });
    
})