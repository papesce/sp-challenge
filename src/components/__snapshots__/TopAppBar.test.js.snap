// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar tests renders correctly 1`] = `
<div>
  <header
    className="MuiPaper-root-8 MuiPaper-shadow4-14 MuiAppBar-root-1 MuiAppBar-positionStatic-4 MuiAppBar-colorPrimary-6"
  >
    <div
      className="MuiToolbar-root-35 MuiToolbar-gutters-36"
    >
      <h2
        className="MuiTypography-root-37 MuiTypography-title-43 MuiTypography-colorInherit-56"
      >
        Spotify Challenge
      </h2>
    </div>
  </header>
</div>
`;
